recipes/conda-create-no-user-site/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: conda-create-no-user-site
+  version: 0.1.0
+
+source:
+  path: .
```

Comment:
What's the maintenance model for this package? Feedstocks are not usually equipped to deal with release processes and PRs, so maybe it's a better idea to commit this to a separate repo (maybe under conda-incubator, if you are inclined). I like seeing more conda plugins in the world and it would help with visibility.